### ISSUE TITLE
Lazy Module 

### ISSUE THREAD

**Issue by tomadj** [thomasduft/blader#1]
Hello @thomasduft ! 
Very good project ! 

The last angular 7 update doesn't resole lazy module issue ? 

Thank's

**Comment by thomasduft** [comment]
Salut @tomadj 
No unfortunately not. But this is by design of how the injector and creating child injectors works. I upgraded to the latest ng versions and reenabled lazyBlade.module by default. If you now hit the "lazy" route you will get an error that looks similar to the following:
```javascript
BladerComponent.html:2 ERROR Error: No component factory found for LazyBladeComponent. Did you add it to @NgModule.entryComponents?
    at noComponentFactoryError (core.js:7754)
    at CodegenComponentFactoryResolver.push../node_modules/@angular/core/fesm5/core.js.CodegenComponentFactoryResolver.resolveComponentFactory (core.js:7792)
    at BladeComponent.push../src/app/blader/blade.component.ts.BladeComponent.ngOnInit (blade.component.ts:79)
    at checkAndUpdateDirectiveInline (core.js:18620)
    at checkAndUpdateNodeInline (core.js:19884)
    at checkAndUpdateNode (core.js:19846)
    at debugCheckAndUpdateNode (core.js:20480)
    at debugCheckDirectivesFn (core.js:20440)
    at Object.eval [as updateDirectives] (BladerComponent.html:2)
    at Object.debugUpdateDirectives [as updateDirectives] (core.js:20432)
```
Which makes sense because the lazy blade component lives within the child injector. The application but tries to locate the component within the root injector and logically cannot find it.

The solution to this problem would be to make the root injector aware of the things that live within the child injector. If I find time I will try to create a service that solves this issue. 
Personally I don't think that the angular community will fix this issue because of how the injector system works.

Bonne journée
Thomas

**Comment by thomasduft** [comment]
resolved by #2.

**Comment by tomadj** [comment]
Thank you so much ! 
I will test that !!
